temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
+        else:
+            log.error("ERROR: Invalid file type")
```

Comment:
**Suggestion:** Handle the case where the file at the given filepath does not exist or is not writable. [error handling]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
import os

def reset_files(filepath, file_type="txt"):
    if not os.path.exists(filepath):
        log.error("ERROR: File does not exist")
        return
    if not os.access(filepath, os.W_OK):
        log.error("ERROR: File is not writable")
        return
    with open(filepath, "w") as output_file:
        if file_type == "txt":
            output_file.write("")
        elif file_type == "json":
            output_file.write(json.dumps({}))
        else:
            log.error("ERROR: Invalid file type")
```